classes/author.rb

```diff
@@ -0,0 +1,17 @@
+require_relative 'book'
+
+class Author
+  attr_reader :id, :first_name, :last_name, :books
+
+  def initialize(first_name, last_name)
+    @id = Random.rand(1..1000)
+    @first_name = first_name
+    @last_name = last_name
+    @books = []
+  end
+
+  def add_book(book)
+    @books << book
+    book.author = self
+  end
```

Comment:
- [ ] I think🤔🤔 it would be better to use the `add_item` method instead of `add_book` . This change would adhere to the project requirements and maintain consistency with the naming convention used throughout the codebase.😍😍
![Screenshot 2023-06-13 174922](https://github.com/AbdusSattar-70/Ruby-Group-Capstone/assets/118761303/eaa53665-d10f-48e6-aea3-afa83b2d3645)